### DISCUSSION TITLE
Missing column in dump file

### DISCUSSION THREAD

**Discussion by samujob** [mydumper/mydumper#623] [General]
- I am using mydumper to dump one table from a database
- The database that I am dumping from is a replica (I have setup replication from the master to this database)
- The sql_mode has ALLOW_INVALID_DATES
- This table has a composite primary key - say col1 (varchar), col2 (datetime) and col3 (varchar)
- The col2 (datetime) has default value CURRENT_TIMESTAMP

When I restored the dump on the target, I found that all values in col2 were default value generated by CURRENT_TIMESTAMP. I then investigated the dump file and found that the col2 is not in the insert statement at all.

Any idea as to why mydumper would not include this column in the dump?

**Comment by davidducos** [comment]
This might be a bug, please, share a full test case and can you move this to Issues??
